File: Programming Basics with JS/T08 - Exam Preparation/lecture/05. Suitcases Load.js
function suitcases(input) {
    let trunkSpace = Number(input[0]);
    let index = 1;
    command = input[index];

    let counter = 0;
    let totalSuitcases = 0;

    while (command !== 'End') {
        counter++;
        let suitcase = Number(command); // 100

        if (counter === 3) {
            suitcase = suitcase + (suitcase * 10 / 100);
            counter = 0;
        }
        trunkSpace -= suitcase;

        if (trunkSpace < 0) {
            console.log(`No more space!`);
            break;
        }
        totalSuitcases++;

        index++;
        command = input[index];
    }

    if (trunkSpace >= 0) {
        console.log(`Congratulations! All suitcases are loaded!`);
    }
    console.log(`Statistic: ${totalSuitcases} suitcases loaded.`);
}
suitcases(["1200.2", "260", "380.5", "125.6", "305", "End"]);

File: Programming Basics with Python/T08 - Exam Preparation/lecture/05. Suitcases Load.py
capacity = float(input())
volume = input()
cases_used_volume = 0
left_space = capacity
cases = 0

while volume != "End":
    cases += 1
    volume = float(volume)

    if cases % 3 == 0:
        volume = volume + (volume * 0.1)
    cases_used_volume += volume
    left_space -= volume

    if capacity < cases_used_volume:
        print("No more space!")
        print(f"Statistic: {cases - 1} suitcases loaded.")
        exit()
    volume = input()

if capacity >= cases_used_volume:
    print("Congratulations! All suitcases are loaded!")
    print(f"Statistic: {cases} suitcases loaded.")